src/app/static/src/app/store/stepper/actions.ts

```diff
@@ -0,0 +1,38 @@
+import {ActionContext, ActionTree} from "vuex";
+import {RootState} from "../../root";
+import {StepDescription, StepperState} from "./stepper";
+import {localStorageManager} from "../../localStorageManager";
+
+export interface StepperActions {
+    jump: (store: ActionContext<StepperState, RootState>, step: number) => void,
+    next: (store: ActionContext<StepperState, RootState>) => void
+}
+
+export const actions: StepperActions & ActionTree<StepperState, RootState> = {
+    jump({commit}, step) {
+        commit({type: "Jump", payload: step});
+    },
+
+    next(store) {
+        const {state, getters} = store;
+        if (getters.complete[state.activeStep]) {
```

Comment:
Btw seems like it's perfectly good practice to access getters from an action - wasn't this what was needed in the filter flow to update the selected filter option when the selected data type changed?

Reply:
Hm. I think i was looking at the mutations for that rather than the actions, but maybe it would have made most sense in the actions anyway...